pkg/operator/ceph/csi/controller.go

```diff
@@ -194,6 +194,13 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 		return opcontroller.ImmediateRetryResult, errors.Wrap(err, "failed to create pool ID mapping config map")
 	}
 
+	exists, err := checkCsiCephConfigMapExists(r.opManagerContext, r.context.Clientset, r.opConfig.OperatorNamespace)
+	if err != nil {
+		return opcontroller.ImmediateRetryResult, errors.Wrap(err, "failed to get csi ceph.conf configmap")
+	}
+	if exists {
+		CustomCSICephConfigExists = true
```

Comment:
How is the configmap content being configured?

Reply:
The user needs to set the configuration in the csi configmap, if the configmap is present Rook will mount the file to the cephcsi container to `/etc/ceph/ceph.conf`.

Reply:
How about always setting the value, even if it doesn't exist?
```suggestion
		CustomCSICephConfigExists = exists
```